clang/lib/AST/Decl.cpp

```diff
@@ -35,6 +35,7 @@
 #include "clang/AST/Stmt.h"
 #include "clang/AST/TemplateBase.h"
 #include "clang/AST/Type.h"
+#include "clang/AST/TypeBase.h"
```

Comment:
Shouldn't need to include TypeBase, that's an internal header, and since Type.h is already included anyway.
```suggestion
```